score/score_test.go

```diff
@@ -307,3 +306,37 @@ func TestList(t *testing.T) {
 	assert.True(t, hasService)
 	assert.True(t, hasDeployment)
 }
+
+func TestPodContainerStorageEphemeralNoLimit(t *testing.T) {
+	t.Parallel()
+	testExpectedScore(t, "pod-ephemeral-storage-missing-limit.yaml", "Container Ephemeral Storage Request and Limit", scorecard.GradeCritical)
+}
+
+func TestPodContainerStorageEphemeralNoRequest(t *testing.T) {
+	t.Parallel()
+	testExpectedScore(t, "pod-ephemeral-storage-missing-request.yaml", "Container Ephemeral Storage Request and Limit", scorecard.GradeWarning)
```

Comment:
Could we have a test that checks for "Container Ephemeral Storage Request and Limit" returning GradeOK?

Reply:
Additional test case (GradeAllOK) added.